sds_data_manager/lambda_code/SDSCode/instruments/l1c_codice.py

```diff
@@ -24,6 +25,23 @@ def lambda_handler(event: dict, context):
     # Get the environment variables
     bucket = os.environ["S3_BUCKET"]
     prefix = os.environ["S3_KEY_PATH"]
+    secret_name = os.environ['SECRET_NAME']
```

Comment:
Rather than putting this secret info in the lambda handler, we should try and connect it directly to the lambda while we're building it. It looks like AWS provides a lambda option with RDS, so that might work. But we shouldn't put this info into the lambda itself if we can avoid it, I think. https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/rds-lambda-tutorial.html

Reply:
I think we are creating secret at CDK build time(I am unsure of where it happens). It gets passed as env variable to the lambda at CDK build time. Then we are referencing env variable here to get below information to connect to RDS. I think you both are thinking similar thing if I understood correctly.

This is from link Maxine shared:
```
# rds settings
user_name = os.environ['USER_NAME']
password = os.environ['PASSWORD']
rds_proxy_host = os.environ['RDS_PROXY_HOST']
db_name = os.environ['DB_NAME']
```
This is from Laura's code:
```
    secret_object = json.loads(secret_value_response['SecretString'])
    secret_object["host"]
    secret_object["username"]
    secret_object["dbname"]
    secret_object["password"]
```